### PR TITLE
Kickstart cartesi-rollups-cli

### LINKED ISSUE #151: Kickstart cartesi-rollups-cli
## 📚 Context

It is hard for the node developer to test the rollups-node because we rely on the rollups-examples infrastructure.

## ✔️ Solution

Having a CLI binary to help manually test the node would be nice.

## 📈 Subtasks

- [x] Create a new Go binary under `cmd`.
- [x] Create the rollups contract bindings in Go.
- [x] Create helper pkg to send transactions using go-ethereum.
- [x] Add command to send inputs.

### PR DESCRIPTION
**Pull request by gligneul**
Close #151

### CODE REVIEW

**Review by renan061 — COMMENTED** [review]
LGTM. But I would like a in-call explanation. Hehehe.

File: docs/cartesi-rollups-cli.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## cartesi-rollups-cli
+
+Command line interface for Cartesi Rollups
+
+### Synopsis
+
+Command line interface for Cartesi Rollups
```
Comment (gligneul) [inline docs/cartesi-rollups-cli.md:7]: Repeated

File: pkg/ethutil/ethutil.go [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	return 0, fmt.Errorf("input index not found")
+}
+
+// Get the given input of the given DApp from the input box.
+// Return the input sender and the input payload.
+func GetInputFromInputBox(
+	client *ethclient.Client,
+	addresses *addresses.Book,
+	inputIndex int,
+) (common.Address, []byte, error) {
```
Comment (gligneul) [inline pkg/ethutil/ethutil.go:91]: This function should return the event

**Comment by GMKrieger** [comment]
I'm with Renan, LGTM but would like further explaining in call.

**Review by GMKrieger — DISMISSED** [review]

**Review by marcelstanley — DISMISSED** [review]

**Review by torives — COMMENTED** [review]
Just left some minor suggestions. I loved the auto-generated docs and bindings, didn't know go:generate existed ♥️

**Review by torives — DISMISSED** [review]

File: pkg/ethutil/ethutil_test.go [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+	"github.com/stretchr/testify/require"
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+const testTimeout = 300 * time.Second
+
+// Test sending inputs to the input box.
+// This function test the GetInputFromInputBox function as well.
+func TestSendingInputs(t *testing.T) {
```
Comment (gligneul) [inline pkg/ethutil/ethutil_test.go:25]: I will change this test to use testify Suite lib.

**Review by marcelstanley — DISMISSED** [review]
Just a minor change and we're good

**Review by marcelstanley — APPROVED** [review]

**Review by GMKrieger — APPROVED** [review]

**Pull request merged**